calico-enterprise_versioned_docs/version-3.15/operations/troubleshoot/commands.mdx

```diff
@@ -237,8 +237,8 @@ items:
 - apiVersion: operator.tigera.io/v1
   kind: Installation
   metadata:
-    - apiVersion: operator.tigera.io/v1
- spec:
+    name: default
```

Comment:
This seems to be a change. Do we need to get SME review for this?

Reply:
Fixed differently.